### PR TITLE
Create Section 2 'licensing' of Controller UG

### PR DESCRIPTION
**Pull request by ianf77**
Added files and assembly. Updated maaster.adoc

Create Section 2 of automation controller user guide

https://issues.redhat.com/browse/AAP-15344

### CODE REVIEW

File: downstream/assemblies/platform/assembly-controller-licensing.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-controller-licensing"]
+= {ControllerNameStart} licensing, updates and support
+
+:context: licensing-controller
+
+{PlatformName} controller ("Automation Controller") is a software product provided as part of an annual {PlatformName} subscription entered into between you and Red Hat, Inc. ("Red Hat").
```
Comment (ariordan-redhat) [inline downstream/assemblies/platform/assembly-controller-licensing.adoc:8]:
```suggestion
{PlatformName} controller ("{ControllerName}") is a software product provided as part of an annual {PlatformName} subscription entered into between you and Red Hat, Inc. ("Red Hat").
```

File: downstream/modules/platform/ref-controller-license-support.adoc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[id="ref-controller-license-support"]
+
+= Support
+Red Hat offers support to paid {PlatformName} customers.
+
+If you or your company has purchased a subscription for {PlatformNameShort}, you can contact the support team at https://access.redhat.com
```
Comment (ariordan-redhat) [inline downstream/modules/platform/ref-controller-license-support.adoc:6]:
```suggestion
If you or your company has purchased a subscription for {PlatformNameShort}, you can contact the support team at https://access.redhat.com.
```

File: downstream/modules/platform/ref-controller-trial-evaluation.adoc [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[id="ref-controller-trial-evaluation"]
+
+= Trial and evaluation
+You require a license to run {ControllerName}, however, there is no fee for a trial license.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/ref-controller-trial-evaluation.adoc:4]:
```suggestion

You require a license to run {ControllerName}. However, there is no fee for a trial license.
```

File: downstream/modules/platform/ref-controller-trial-evaluation.adoc [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+Trial licenses for {PlatformNameShort} are available at: http://ansible.com/license.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/ref-controller-trial-evaluation.adoc:6]: Should http://ansible.com/license be a link?

File: downstream/modules/platform/ref-controller-node-counting.adoc [hunk 5]
```diff
@@ -0,0 +1,14 @@
+[id="ref-controller-node-counting"]
+
+= Node counting in licenses
+
+The {ControllerName} license defines the number of Managed Nodes that can be managed as part of a {PlatformName} subscription.
+
+A typical license says 'License Count: 500', which sets the maximum number of Managed Nodes at 500.
+
+For more information on managed node requirements for licensing, see https://access.redhat.com/articles/3331481.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/ref-controller-node-counting.adoc:9]: Should https://access.redhat.com/articles/3331481 be a link?

**Review by ariordan-redhat — COMMENTED** [review]
Minor, ignorable suggestions.

File: downstream/modules/platform/ref-controller-license-support.adoc (hunk 2 above)
Comment (ianf77) [reply downstream/modules/platform/ref-controller-license-support.adoc:6]: Tried adding a full stop here, it becomes part of the URL and hence might fail.

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**